website/www/site/static/js/hero/hero-desktop.js

```diff
@@ -0,0 +1,19 @@
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+var animation = bodymovin.loadAnimation({
+  container: document.getElementById("hero-desktop"),
+  renderer: "svg",
+  loop: false,
+  autoplay: true,
+  path: "/js/hero/hero-desktop.json",
```

Comment:
Does this work if you just specify a relative url, `hero-desktop.json`? As it is it doesn't work on the staged preview since it's in a sub-directoty: http://apache-beam-website-pull-requests.storage.googleapis.com/13439. It tries to access http://apache-beam-website-pull-requests.storage.googleapis.com//js/hero/hero-desktop.json

Alternatively we might inline the JSON with the `animationData` param, but as noted before that has its own drawback.

This wouldn't be a problem in production of course, but it would be nice to keep the preview working.

Reply:
Hmm, thanks for your suggestion about `animationData`. I agree that maybe that has its own drawback. However, this approach solves 2 questions (license headers of json files & staging deployment). Then, I'd go for it. 😄

Reply:
LG, thanks!